### PR TITLE
v0.14.0 CIRC-9164

### PR DESCRIPTION
**Pull request by maier**
* feat: add node_name tag for pods
* feat: add owner tag for pods
* feat: add support for label_tags=* to turn all labels into tags for the given object

### CODE REVIEW

File: internal/dc/dc.go [hunk 1]
```diff
@@ -430,6 +429,30 @@ func (dc *DC) collectPods(ctx context.Context, collector Collector) {
...
 		tags := generateTags(collector.Tags, collector.LabelTags, item.Labels)
 		tags = append(tags, "collector_target:"+item.Name)
+		if len(item.OwnerReferences) > 0 {
+			var ownerName, ownerKind string
+			switch item.OwnerReferences[0].Kind {
+			case "ReplicaSet":
+				replica, repErr := clientset.AppsV1().ReplicaSets(item.Namespace).Get(ctx, item.OwnerReferences[0].Name, metav1.GetOptions{})
+				if repErr != nil {
+					logger.Warn().Str("pod", item.Name).Msg("unable to get replicaset for pod")
+				} else if len(replica.OwnerReferences) > 1 {
```
Comment (dhaifley) [inline internal/dc/dc.go:439]: Just making sure this is supposed to be Len > 1 not Len > 0, since you are getting the first index in the next line.

**Review by dhaifley — APPROVED** [review]
LGTM

File: internal/dc/dc.go (hunk 1 above)
Comment (maier) [reply internal/dc/dc.go:439]: nope, you're right, that should be a 0... thanks for the catch!

**Review by dhaifley — APPROVED** [review]
LGTM

**Pull request merged**